deployment/common/changeset/internal/mcms.go

```diff
@@ -114,8 +136,14 @@ func DeployMCMSWithTimelockContracts(
 				[]common.Address{canceller.Address, proposer.Address, bypasser.Address}, // cancellers
 				[]common.Address{bypasser.Address},                                      // bypassers
 			)
+
+			tv := deployment.NewTypeAndVersion(types.RBACTimelock, deployment.Version1_0_0)
```

Comment:
Do we maybe want a deployment.NewAddressBookKey(string, string, *string) and then just pass in config.Label (if it's nil, it's nil, that's expected, as it's optional). Then this is a one-liner that is clearer, and doesn't have to have the conditional logic?

That's sort of presaging the rename from TypeAndVersion to AddressBookKey but still...